vue-app/src/views/Join.vue

```diff
@@ -84,17 +84,16 @@
                   <p
                     :class="{
                       error: true,
-                      hidden: !$v.form.project.tagline.$error,
+                      hidden: !$v.form.project.tagline.$error || !$v.form.project.tagline.$model,
                     }"
                   >
                     This tagline is too long. Be brief for potential
                     contributors
                   </p>
-                  <!-- TODO: validation for different error type -->
                   <p
                     :class="{
                       error: true,
-                      hidden: !$v.form.project.tagline.$error,
+                      hidden: !$v.form.project.tagline.$error || !!$v.form.project.tagline.$model,
```

Comment:
Similar to the explanation above, just inverse. Would change to:
` || !$v.form.project.tagline.maxLength`